assets/scss/_variables.scss

```diff
@@ -91,30 +91,11 @@ $containers: (
   }
 }
 
-@mixin scrollbars($scroll-color: rgba(255, 255, 255, 0.2)) {
-  scrollbar-color: $scroll-color transparent;
-  scrollbar-width: thin;
-
-  &::-webkit-scrollbar {
-    height: var(--mm-border-radius);
-    width: var(--mm-border-radius);
-  }
-
-  &::-webkit-scrollbar-thumb {
-    background-color: $scroll-color;
-    border-radius: var(--mm-border-radius);
-  }
-
-  &::-webkit-scrollbar-corner {
-    background: transparent;
-  }
-}
-
 $pattern-scale: 1;
 
 @mixin pattern-base {
```

Comment:
All of these `@mixin` should go into `_mixins.scss`.  The `$variables` can stay.

Reply:
Separating them out creates this error for multiple scss files:

![Screen Shot 2022-06-23 at 6 50 38 PM](https://user-images.githubusercontent.com/52248962/175427781-c7f66b86-b626-43f8-bb84-7159287634ef.png)

I tried changing `@include variables.breakpoint('md') {` to `@include mixin.breakpoint('md') {` since we put all the mixins into a different file, but that didn't work either.

Reply:
Wait a second, are any of the mixins in `_mixin.scss` even being used?

Reply:
Sorry, I didn't properly go through those files when I copied them over from the other sites. I cleaned up and reorganized them in a new commit. I wasn't sure how to separate out the mixins that were used for the breakpoints in `_variables.scss`, though.